wiki/Game_modifier/Auto/id.md

```diff
@@ -0,0 +1,60 @@
+---
+stub: true
+tags:
+  - auto
+  - autoplay
+  - auto play
+  - AT
+  - AO
+  - mod
+  - game modifier
+---
+
+# Auto (mod)
+
+![Ikon mod AT](/wiki/shared/mods/AT.png "Ikon mod Auto (AT)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Autopilot (mod)](/wiki/Game_modifier/Autopilot).*
+
+## Tentang
+
+- Singkatan: AT atau AO
+- Jenis: Spesial
+- Pengali Skor: 1.00x
+- Tombol shortcut: `V`
+- Deskripsi: `Tonton permainan sempurna yang berjalan secara otomatis`<!-- yes, that is written correctly -->
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
```

Comment:
```suggestion
- Mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

Reply:
these are the URL references (a full link for each is usually at the bottom of the article). they are invisible, intentionally made short, and their names do not matter

Reply:
the same goes for other PRs where you left similar suggestions

Reply:
i think this was reference of
![image](https://user-images.githubusercontent.com/39128490/115442583-628bd200-a23c-11eb-8672-587b41857208.png)
that recently changed on FA showcase article